models/system/databricks-dolly-v2-12b/spec.yaml

```diff
@@ -17,4 +17,4 @@ tags:
   Preview: ''
   license: mit
```

Comment:
@ankushbhatia2 as discussed lets change this to apache?

Reply:
anything we need to update in properties specific to finetuning and evaluation min sku's?